pkgs/by-name/li/livi/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, gst_all_1
+, wrapGAppsHook4
+, appstream-glib
+, gtk4
+, libadwaita
+, desktop-file-utils
+, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "livi";
+  version = "0.0.6";
+
+  src = fetchFromGitLab {
+    owner = "guidog";
+    repo = "livi";
+    domain = "gitlab.gnome.org";
+    rev = "v${version}";
+    sha256 = "sha256-DaIbBCJT4Da5noW6Q5z1yzTZ256HNqrvdXgwSY7p/D8=";
+  };
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-good
```

Comment:
These gst packages should be in `buildInputs`

Reply:
Oh. I miss interpreted the previous review and instead of moving the last line, I moved the whole highlighted section into `nativeBuildInput`. I'll fix this.
How do you know which packages should get into `buildInputs`?

Reply:
I should admit that I mostly go by feel. Packages that are used at build time should go in `nativeBuildInputs`, and packages that are linked or called at runtime should go in `buildInputs`. If you want a very accurate representation, you can add `strictDeps = true`, in this way if you don't do it correctly, you'll fail to build.